_typos.toml

```diff
@@ -0,0 +1,20 @@
+# https://github.com/crate-ci/typos#false-positives
+[default]
+
+[default.extend-identifiers]
+Lik = "Lik"
```

Comment:
maybe as in likelihood, `logLik` or something?

Reply:
exactly -- and this arises from the way tokenization happens